src/main/java/frc/robot/RobotContainer.java

```diff
@@ -293,22 +297,31 @@ private void configureButtonBindings() {
             () -> {
               SignalLogger.stop();
             }));
-
+    coralScorer
+        .hasCoralTrigger()
+        .and(elevator.elevatorUpTrigger())
+        .and(coralScorer.scorerAlignedTrigger().negate())
+        .onTrue(signaling.setColor(RGB.YELLOW));
+    coralScorer
+        .hasCoralTrigger()
+        .and(elevator.elevatorUpTrigger())
+        .onFalse(signaling.setColor(RGB.BLUE));
     coralScorer
         .scorerAlignedTrigger()
         .and(coralScorer.hasCoralTrigger())
         .and(elevator.elevatorAtSetpoint(ElevatorConstants.kL0Height).negate())
         .and(elevator.elevatorAtCurrentSetpoint())
         .whileTrue(
-            Commands.runEnd(
+            Commands.startEnd(
                 () -> {
                   OI.Driver.setRumble(0.5);
                   OI.Operator.setRumble(0.5);
                 },
                 () -> {
                   OI.Driver.setRumble(0);
                   OI.Operator.setRumble(0);
-                }));
+                }))
+        .onTrue(signaling.setColor(RGB.GREEN));
```

Comment:
Why not just add this to the startEnd Command? there's no reason to have an onTrue here.